bin/check_tron_jobs.py

```diff
@@ -172,12 +177,32 @@ def get_relevant_action(action_runs, last_state):
     return action_runs[-1]
 
 
+def guess_realert_every(job):
+    try:
+        job_next_run = job.get('next_run', None)
+        job_runs = job.get('runs', [])
+        job_previous_run = max(
+            [run for run in job_runs if run['start_time'] is not None], key=lambda k: k['start_time'],
```

Comment:
Sure, I think it would make sense to alert again if a manual run fails. But I am not sure how - I don't think this logic would necessarily do that.
Maybe it's enough that the alert doesn't resolve? If the manual run succeeds, the alert will resolve as soon as the run is over.

Reply:
If the manual run succeeds, the alert would resolve. If the manual run fails, since it has the most recent start_time, the realert_every would be set up as the smaller number. This might not send out the alert immediately (depending on the timing of the manual run, close to the previous run or close to next_run), but it would send out the alert again before the next run. Well, I am not sure what the best behavior is here, but I guess it can't hurt anything more 🤕

Reply:
I think it could send out the alert before the manual run finishes, which might be confusing?
 Anyway, as long as the email correctly shows which job run is causing the alert, I guess re-alerting won't hurt too much.